test/org/sagebionetworks/bridge/TestUserAdminHelper.java

```diff
@@ -112,9 +112,10 @@ public void deleteUser(TestUser testUser) {
         checkNotNull(testUser);
         
         if (testUser.getSession() != null) {
+            String userId = testUser.getId();
             authService.signOut(testUser.getSession());
+            deleteUser(testUser.getStudy(), userId);
         }
-        deleteUser(testUser.getStudy(), testUser.getId());
     }
```

Comment:
Shouldn't the user be deleted even if they have no session?

Reply:
I've restored this. This is definitely an error.